### PR TITLE
moving qmri estimation scripts to public repo

### PR DESCRIPTION
**Pull request by man-shu**
* all preprocessing functions and t1 and t2 pipelines are in `ibc_public/utils_relaxo.py`
* all qmri package dependent estimation scripts are in `scripts/`
* `scripts/qmri_run_estimation.py` returns estimated t1 and t2-maps in both MNI and subject-space
* `scripts/qmri_README.md` contains details about all the pipelines

### CODE REVIEW

File: scripts/qmri_t1_map_b1_params.py [hunk 1]
```diff
@@ -0,0 +1,118 @@
+##########################################################################
```
Comment (bthirion) [inline scripts/qmri_t1_map_b1_params.py:1]: Same here.

**Comment by bthirion** [comment]
If these are just scripts that call qmri, without providing core functionalities, it is OK to have them. Thx.

**Comment by man-shu** [comment]
These indeed are the scripts that call qmri, but they were a part of the qmri package and were written by people who developed the rest of the package and not me. I just made some modifications like, removing the dicom dependency and FSL masking. So I suppose we would still need their approval.

**Comment by bthirion** [comment]
If it a mere script, don't worry about it.  Thx.

**Review by bthirion — APPROVED** [review]

**Comment by man-shu** [comment]
Ok! Merging...

**Pull request merged**